clients/admin-ui/src/features/common/nav/v2/nav-config.test.ts

```diff
@@ -57,8 +57,8 @@ describe.skip("configureNavGroups", () => {
       config: NAV_CONFIG,
       hasSystems: true,
       userScopes: [
-        ScopeRegistryEnum.CLI_OBJECTS_CREATE,
-        ScopeRegistryEnum.CLI_OBJECTS_READ,
+        ScopeRegistryEnum.SYSTEM_CREATE,
+        ScopeRegistryEnum.SYSTEM_READ,
```

Comment:
I didn't follow too closely the decisions made last week, but I wonder if the latest in `main` would mean we wouldn't have to do this here? might be worth a merge to see if the tests "just pass"

Reply:
Agree, this is not needed after pulling in latest from Dawn's branch.